Tests/js/views/assets/ez-userprofileview-tests.js

```diff
@@ -109,6 +110,49 @@ YUI.add('ez-userprofileview-tests', function (Y) {
             Y.Assert.areSame(this.view, item.getTargets()[0], 'Should set correct event target');
             Y.Assert.isInstanceOf(Y.eZ.UserMenuView, item, "The user menu view should contain an instance of the constructor");
         },
+
+    });
+
+    userMenuDisplayTest = new Y.Test.Case({
+        name: "eZ User Profile attribute test",
+
+        setUp: function () {
+            var View = Y.Base.create('userMenuView', Y.View, [], {}, {
+                    ATTRS: {
+                        displayed: {
+                            value: false,
+                        }
+                    }
+                });
+            this.view = new Y.eZ.UserProfileView({
+                userMenuView: new View(),
+            });
+            this.view.get('userMenuView').addTarget(this.view);
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should add the menu displayed class": function () {
+            this.view.get('userMenuView').set('displayed', true);
+
+            Assert.isTrue(
+                this.view.get('container').hasClass('is-menu-displayed'),
```

Comment:
You use `is-menu-displayed` twice, maybe it's a good idea to define a variable `CLASS_IS_DISPLAYED`? What do you think about it?